docs/conf.py

```diff
@@ -155,9 +156,9 @@
 }
 
 
-def setup(app: Sphinx) -> None:
-    """Setup lamindb for CI."""
-    import lamindb as ln
+# def setup(app: Sphinx) -> None:
```

Comment:
If this code is precent, receive the error
```
You cannot write to different instances in the same Python session.

Do you want to read from another instance via `Record.using()`? For example:

ln.Artifact.using("laminlabs/cellxgene").filter()

Or do you want to switch off auto-connect via `lamin settings set auto-connect false`?
```

Reply:
As long as the docs build on RTD and locally, everything’s fine!

That was my uninformed attempt to make things work.